instrumentation/src/main/java/io/opentelemetry/android/OpenTelemetryRumBuilder.java

```diff
@@ -297,16 +300,36 @@ public OpenTelemetryRum build() {
                         .setPropagators(buildFinalPropagators())
                         .build();
 
+        setUpDiskBuffering();
```

Comment:
So I think this name is wrong. "Buffering" implies sending something into a buffer. From where I stand, this is exactly the opposite of that -- it's reading something that has already been buffered (on disk). A better name might be something like `schedulePeriodicDiskReadingForBufferedTelemetry()` or maybe just `scheduleDiskTelemetryReader()`.

Reply:
Makes sense 👍  it's updated now.